swan-lake/learn-the-language/distinctive-language-features/data.md

```diff
@@ -560,7 +560,7 @@ Ballerina's philosophy is to use tables as containers for building centralized d
 
 Tables are a built-in data structure. They are just like the arrays and maps that you have seen so far. Therefore, they have some array-like and some map-like features.
 
-A table is an array of records, and each record represents a row in the table. The rows are identified by keys, which is similar to maps. Thus, you can either iterate over the table, item by item, like arrays, or directly point to the item using the associated key. But unlike in maps where the keys are of ``string`` type and are different from the fields, a table stores the keys as fields in the rows. This approach is similar to the concept of primary keys in a SQL-based database table where one of the columns is designated as a primary key, which is used to uniquely identify the database record.
+A table is an array of records and each record represents a row in the table. The rows are identified by keys, which are similar to maps. Thus, you can either iterate over the table item by item like in arrays, or directly point to the item using the associated key. However, unlike in maps where the keys are of the ``string`` type and are different from the fields, a table stores the keys as fields in the rows. This approach is similar to the concept of primary keys in an SQL-based database table in which one or a combination of a few columns are designated as a primary key, which is used to uniquely identify the database record.
```

Comment:
```
The rows are identified by keys, which is similar to maps.
```

Here what is similar to maps is how members are identified - by key. Not that keys are similar to maps.

```suggestion
A table is an array of records and each record represents a row in the table. The rows are identified by keys, similar to maps. Thus, you can either iterate over the table item by item like with arrays, or directly point to the item using the associated key. However, unlike in maps where the keys are of the ``string`` type and are different from the fields, a table stores the keys as fields in the rows. This approach is similar to the concept of primary keys in an SQL-based database table in which one column or a combination of a few columns are designated as a primary key, which is used to uniquely identify the database record.
```